stockflux-securities-master/src/components/AddSecurityButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./AddSecurityButton.css";
+
+const AddSecuritiesButton = ({size}) => {
```

Comment:
Can you add some spacing around the parameters passed in, `({ size })`?

Reply:
done